packages/transpiler/src/index.ts

```diff
@@ -88,8 +102,11 @@ export class Transpiler {
         return transpiler.transpile(node);
       }
     }
-
-    return "todo, statement: " + node.get().constructor.name;
+    if (this.options.checks === CHECKS.comment) {
+      return "todo, statement: " + node.get().constructor.name;
+    } else {
+      throw new Error(`Statement ${node.get().constructor.name} not supported`);
```

Comment:
@larshp I added options  to the transpiler to react on that as per the wishes of the consumer

Reply:
think its okay to only just throw the error

Reply:
I think its okay not to have any options, and just throw the exception here

Reply:
I can imagine having options (particularly checks) as valuable in future. Does it hurt, shall I remove it?

Reply:
yes, keep the throw, remove the options

there will be options, but its always a mess to have multiple behaviors